src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from '../../types/types';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const isSelectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const HandleCLick = (tab: Tab) => {
```

Comment:
This is not handle for click. You use this function inside your event handler
```suggestion
  const selectTab = (tab: Tab) => {
```